core/storage/statistics/gae_models_test.py

```diff
@@ -43,16 +46,189 @@ def test_create_and_get_event_models(self):
         self.assertEqual(event_model.is_feedback_useful, True)
 
 
+class StateCounterModelTests(test_utils.GenericTestBase):
+
+    def test_stat_model_gets_created(self):
```

Comment:
Here and in other test names: stats_model, not stat_model. (Or, better still, state_counter_model.)

Reply:
Done